### PR TITLE
blacklisted various program files

### PR DESCRIPTION
**Pull request by valoq**
- added several directories to disable-programs.inc
- fixed konsole issue #950

### CODE REVIEW

File: etc/disable-programs.inc [hunk 1]
```diff
@@ -44,7 +44,27 @@ blacklist ${HOME}/.openshot_qt
 blacklist ${HOME}/.flowblade
 blacklist ${HOME}/.config/flowblade
 blacklist ${HOME}/.config/eog
-
+blacklist ${HOME}/.config/arkrc
+blacklist ${HOME}/.config/atril
```
Comment (Fred-Barclay) [inline etc/disable-programs.inc]: Aren't a lot of these duplicates? For example, 
`blacklist ${HOME}/.config/atril` is on lines 20 and 48
`blacklist ${HOME}/.config/aweather` is on 18 and 49
`blacklist ${HOME}/.config/Cryptocat` is on 53 and 137
`blacklist ${HOME}/.local/share/gnome-chess` is on 146 and 194

There may be some others but that's all I noticed at first glance.

File: etc/gnome-chess.profile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Firejail profile for gnome-chess
-noblacklist /.local/share/gnome-chess
+noblacklist ~/.local/share/gnome-chess
```
Comment (Fred-Barclay) [inline etc/gnome-chess.profile]: Good catch!

File: etc/disable-programs.inc (hunk 1 above)
Comment (Fred-Barclay) [inline etc/disable-programs.inc]: Aren't a lot of these duplicates? For example, 
`blacklist ${HOME}/.config/atril` is on lines 20 and 48
`blacklist ${HOME}/.config/aweather` is on 18 and 49
`blacklist ${HOME}/.config/Cryptocat` is on 53 and 137
`blacklist ${HOME}/.local/share/gnome-chess` is on 146 and 194

There may be some others but that's all I noticed at first glance.

**Comment by SpotComms** [comment]
Feel free to diff against my [disable-programs.inc](https://github.com/SpotComms/FirejailProfileGenerator/blob/master/generated/disable-programs.inc), I haven't updated it since before https://github.com/netblue30/firejail/commit/fa10ab0e093a4224b16491273b0162b0e0a77a3a , so its still missing a few programs.

File: etc/disable-programs.inc (hunk 1 above)
Comment (valoq) [inline etc/disable-programs.inc]: Thanks, fixed now.

**Comment by valoq** [comment]
Thanks for the list.

I included the missing program files and restructured the list alphabetically for easier maintenance.
Some files that are included in disable-common are not included here

**Comment by netblue30** [comment]
all merged, thanks.

**Pull request merged**